packages/caravan-psbt/src/__tests__/functions.test.ts

```diff
@@ -1,4 +1,5 @@
-import { bufferize } from "./functions";
+import { bufferize } from "../functions";
+import { describe, it, expect } from "vitest";
```

Comment:
Shouldn't this not be necessary if the globals are setup properly?

Reply:
Yeah correct, thank you suggestions I've removed those unnecessary imports